src/main/js/actions/grades.js

```diff
@@ -0,0 +1,14 @@
+import {
+  ADD_COURSEGPA,
+  SELECT_SORTBYGPA,
+} from './types';
+
+export const addGrades = gpa => ({
+  type: ADD_COURSEGPA,
+  gpa,
+});
+
+export const sortByGPA = sort => ({
+  type: SELECT_SORTBYGPA,
```

Comment:
lets call this `SELECT_SORT`, also the function name should mirror the action. Also this action is pretty `generator` specific so should move it to that file (same with reducer).

Reply:
Done.